### PR TITLE
Script to run tutorials on the CI

### PR DESCRIPTION
**Pull request by syarra**
**I hereby confirm that I have:**

- [x] Tested my code on my own machine
- [x] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [x] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [ ] with [X] to check the box)*

### CODE REVIEW

File: .ci/runTutorial.m [hunk 1]
```diff
@@ -0,0 +1,24 @@
+function runTutorial(tutorialName)
+
```
Comment (laurentheirendt) [inline .ci/runTutorial.m]: May you add some documentation here?

File: .ci/runTutorial.m [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+% include the root folder and all subfolders
+addpath(genpath(pwd))
```
Comment (laurentheirendt) [inline .ci/runTutorial.m]: Is it necessary to add the entire path? What about hidden folders? 

Unless I am mistaken, it should be enough to only add the root itself, i.e. `addpath(pwd)`. The remaining paths are added when running `initCobraToolbox`.

File: .ci/runTutorial.m [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+% run the official initialisation script
+initCobraToolbox
+
+% Mute progress bars and initCobraToolbox
+global WAITBAR_TYPE;
```
Comment (laurentheirendt) [inline .ci/runTutorial.m]: May you declare the `global` variables further up?

File: .ci/launchTutorials.sh [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+report="Tutorial report\n\n"
+report+="Name                               passed    failed    time(s)\n"
+report+="--------------------------------------------------------------\n"
+failure=0
+
+# Set time format to seconds
+TIMEFORMAT=%R
+
+for tutorial in "${tutorials[@]}"
```
Comment (laurentheirendt) [inline .ci/launchTutorials.sh]: 🎉

File: .ci/runTutorial.m (hunk 1 above)
Comment (syarra) [inline .ci/runTutorial.m]: Done!

File: .ci/runTutorial.m (hunk 2 above)
Comment (syarra) [inline .ci/runTutorial.m]: You're right. I changed it.

File: .ci/runTutorial.m (hunk 3 above)
Comment (syarra) [inline .ci/runTutorial.m]: Moved after docstrings

**Comment by laurentheirendt** [comment]
Thanks @syarra 🌵

**Pull request merged**